discord/message.go

```diff
@@ -181,20 +181,28 @@ type Sticker struct {
 	// ID is the ID of the sticker.
 	ID StickerID `json:"id"`
 	// PackID is the ID of the pack the sticker is from.
-	PackID StickerPackID `json:"pack_id"`
+	PackID StickerPackID `json:"pack_id,omitempty"`
 	// Name is the name of the sticker.
 	Name string `json:"name"`
 	// Description is the description of the sticker.
 	Description string `json:"description"`
 	// Tags is a comma-delimited list of tags for the sticker. To get the list
 	// as a slice, use TagList.
 	Tags string `json:"-"`
-	// Asset is the sticker's assert hash.
-	Asset Hash `json:"asset"`
+	// The type of sticker
+	Type StickerType `json:"type"`
 	// PreviewAsset is the sticker preview asset hash.
 	PreviewAsset Hash `json:"preview_asset"`
 	// FormatType is the type of sticker format.
 	FormatType StickerFormatType `json:"format_type"`
+	// Whether this guild sticker can be used, may be false due to loss of Server Boosts
+	Available bool `json:"available,omitempty"`
+	// ID of the guild that owns this sticker
+	GuildID GuildID `json:"guild_id,omitempty"`
+	// The user that uploaded the guild sticker
+	User User `json:"user,omitempty"`
```

Comment:
```suggestion
	// User is the user that uploaded the guild sticker
	User *User `json:"user,omitempty"`
```
Go comments always start with the name of the entity.
Also, since this is a struct with the `,omitempty` tag, it should be pointerized.